tensorflow_model_optimization/python/core/quantization/keras/quantize.py

```diff
@@ -361,6 +376,11 @@ def _quantize(layer):  # pylint: disable=missing-docstring
     if not quantize_config and quantize_registry.supports(layer):
       quantize_config = quantize_registry.get_quantize_config(layer)
 
+    # (experimental) enable sparsity preserved quantization
```

Comment:
To discuss regarding the API with Pulkit. I'm inclined to keep this experimental functionality separate from the existing stable quantization-aware-training APIs, until there has been an actual demonstrated value for it (e.g. some model where pruning + sparsity-preserving quantization-aware training does significantly better than pruning + post-training quantization). 

This logic can be refactored to minimize code duplication while preserving readability of the overall flow.

Some of the issues (BN + Concat) in https://github.com/tensorflow/model-optimization/issues/513#issuecomment-673603923 haven't been addressed either, for getting something checked into this stable code.

Reply:
Regarding (BN + Concat):  current implementation, pqat only update weights quantizers after quantize transform, thus cross-layer modifications remains.

Reply:
> Regarding (BN + Concat): current implementation, pqat only update weights quantizers after quantize transform, thus cross-layer modifications remains. Am I missing something?

No you're not missing something. It's a non-issue.